CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# Contributing to Miden
+
+#### First off, thanks for taking the time to contribute!
+
+You can find a detailed explanation of main project concepts in [docs](https://maticnetwork.github.io/miden/).
+
+We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a [bug](https://github.com/maticnetwork/miden/issues/new)
+- Take part in [discussions](https://github.com/maticnetwork/miden/discussions)
+- Submitting a fix
```

Comment:
nit: It feels like this should have a link as well, since all the others do. Maybe we should link to the pull requests page? Not a big deal though, so whatever you prefer is fine.

Reply:
I'm thought about that too. We have no link to really fit this point, maybe issue creation page link is closest one, because any fix will be firstly proposed through new issue. But having 3/4 bullet points with link to issue creation page will look strange. So, I'm added link to the pull requests page for visual consistency.